rules/windows/command_and_control_iexplore_via_com.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2020/11/28"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/28"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies instances of Internet Explorer (IEXPLORE) started via the Component Object Model (COM) making unusual network
+connections. Adversaries could abuse IE via COM to avoid suspicious processes making network connections and bypass Host
+Firewall restrictions.
```

Comment:
```suggestion
Identifies instances of Internet Explorer (iexplore) being started via the Component Object Model (COM) making unusual network
connections. Adversaries could abuse Internet Explorer via COM to avoid suspicious processes making network connections and bypass host-based
firewall restrictions.
```

Reply:
may need to re-lint